### PR TITLE
test: group poa admin

### LINKED ISSUE #61: Consensus failure when `manifest.Params` doesn't contain `params.Inflation`
Updating the `manifest` module parameters without supplying `Inflation` leads to a consensus failure.

The culprit is
```go
	if !params.Inflation.AutomaticEnabled {
		k.Logger().Debug("Automatic inflation is disabled")
		return nil
	}
```
in `BeginBlocker`

We should add a check.

CC @Reecepbcups
CC @chalabi2

### PR DESCRIPTION
**Pull request by fmorency**
Add Group Proposal tests for when POA is a group address (group admin is the group policy itself).

Test
- Software upgrade plan scheduling and canceling
- Manifest module parameters update
  - All parameters
  - Inflation == nil
  - Empty parameters
- Manifest stakeholders payout (mint) and burn (relates #65)
- POA parameters update
  - All parameters
  - Empty admins
- Bank send
  - Valid sender
  - Invalid sender
- Token factory
  - Token create
  - Token modify metadata
  - Token mint and mint to
  - Token burn and burn from
  - Token force transfer
  - Token admin change
  
This PR is becoming huge so I want to get those merged. We can add more tests later.

Fixes #61

### CODE REVIEW

File: interchaintest/helpers/group.go [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+}
+
+func ExecGroupProposal(ctx context.Context, chain *cosmos.CosmosChain, config *ibc.ChainConfig, accAddr, proposalId string) (string, error) {
+	execCommand := []string{
+		"group", "exec", proposalId,
+	}
+	return exec(ctx, chain, config, chain.GetNode().TxCommand(accAddr, execCommand...))
+}
+
+func exec(ctx context.Context, chain *cosmos.CosmosChain, config *ibc.ChainConfig, command []string) (string, error) {
```
Comment (Reecepbcups) [inline interchaintest/helpers/group.go:87]: This is essentially a wrapper around chainnode.ExecTx()

note to self for upstream: use a pattern which allows us to inject logic on a queried CosmosTx{} type. This way we can extend it's usage more, specifically for Group right away. Same for POA

File: interchaintest/poa_group_test.go [hunk 2]
```diff
@@ -0,0 +1,735 @@
+package interchaintest
```
Comment (Reecepbcups) [inline interchaintest/poa_group_test.go:1]: Make `ictest-group-poa` is not in the `.github/workflows/e2e.yml` file

**Review by Reecepbcups — APPROVED** [review]
pre-approval once e2e is in CI and passes

File: interchaintest/helpers/group.go (hunk 1 above)
Comment (fmorency) [reply interchaintest/helpers/group.go:87]: Absolutely, BUT `chainnode.ExecTx()` doesn't handle `config.Env` which I need.

**Comment by fmorency** [comment]
I'll re-run CI tomorrow, the test is failing because we reached the maximum number of Docker requests.

**Pull request merged**